docs/pages/installation/eks.md

```diff
@@ -6,7 +6,20 @@ permalink: installation/eks
 
 # Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
-## Managed Kubernetes
+Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
```

Comment:
This is a little ambiguous, can this be made a little more detailed like start meshery-ui, log in through the provider, go to `localhost:3000` & then download the token?